recipe/meta.yaml

```diff
@@ -72,7 +74,7 @@ requirements:
     # parquet, feather
     - pyarrow >=10.0.1
     # hdf5
-    - blosc >=1.21.3
+    - blosc >=1.20.1
```

Comment:
It's `blosc >= 1.21.3` here https://pandas.pydata.org/pandas-docs/version/2.3.2/getting_started/install.html#other-data-sources